sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/api/Azure.ResourceManager.RedisEnterprise.netstandard2.0.cs

```diff
@@ -159,11 +165,52 @@ public partial class RedisEnterprisePrivateEndpointConnectionResource : Azure.Re
 }
 namespace Azure.ResourceManager.RedisEnterprise.Models
 {
+    public partial class Capability
+    {
+        internal Capability() { }
+        public string Name { get { throw null; } }
+        public bool? Value { get { throw null; } }
+    }
+    public partial class ClusterPropertiesEncryptionCustomerManagedKeyEncryption
+    {
+        public ClusterPropertiesEncryptionCustomerManagedKeyEncryption() { }
+        public Azure.ResourceManager.RedisEnterprise.Models.ClusterPropertiesEncryptionCustomerManagedKeyEncryptionKeyIdentity KeyEncryptionKeyIdentity { get { throw null; } set { } }
+        public System.Uri KeyEncryptionKeyUri { get { throw null; } set { } }
+    }
+    public partial class ClusterPropertiesEncryptionCustomerManagedKeyEncryptionKeyIdentity
+    {
+        public ClusterPropertiesEncryptionCustomerManagedKeyEncryptionKeyIdentity() { }
+        public Azure.ResourceManager.RedisEnterprise.Models.CmkIdentityType? IdentityType { get { throw null; } set { } }
+        public string UserAssignedIdentityResourceId { get { throw null; } set { } }
```

Comment:
This property should be an arm-id (I am pretty sure about this, but please double confirm)
If it is, please change its type to `ResourceIdentifier` by assigning `arm-id` format to it.

Reply:
Will confirm on this

Reply:
@revanthballa1188 this is correct, UserASsignedIdentityResourceId is an `arm-id`.

Reply:
Is this something we need to change in the swagger spec or is this a concept in the SDK?

This was copied as-is from: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#customer-managed-key-encryption

If there's something that needs to be fixed in the swagger spec, then we should request to change that doc.

Reply:
changed the time to ResourceIdentifier